CommDLL.pas

```diff
@@ -1,46 +1,46 @@
-unit CommDLL;
-interface
+unit CommDLL;
+interface
 uses
   Windows, System.SysUtils, ExtAIAPI, InterfaceDelphi, CommonDataTypes;
-
-const
-  MAX_HANDS = 12;
-
-type
-  TInitDLL = procedure(); StdCall;
-  TTerminDLL = procedure(); StdCall;
-  TInitNewExtAI = procedure(aID: ui8; aActions: IActions) StdCall;
-  TNewExtAI = function(): IEvents; SafeCall; // Same like StdCall but allows exceptions
-
-  TCallback1 = function(sig: ui8): b of object; StdCall;
-
-  // Communication with 1 physical DLL with using exported methods.
-  // Main targets: initialization of DLL, creation of ExtAIs and termination of DLL and ExtAIs
-  TCommDLL = class
-  private
-    fLibHandle: THandle;
-    fDLLPath: String;
-    fExtAIAPICnt: ui8;
-    fExtAIAPI: array [0..MAX_HANDS-1] of TExtAIAPI;
-    fOnInitDLL: TInitDLL;
+
+const
+  MAX_HANDS = 12;
+
+type
+  TInitDLL = procedure(); StdCall;
+  TTerminDLL = procedure(); StdCall;
+  TInitNewExtAI = procedure(aID: ui8; aActions: IActions) StdCall;
+  TNewExtAI = function(): IEvents; SafeCall; // Same like StdCall but allows exceptions
+
+  TCallback1 = function(sig: ui8): b of object; StdCall;
+
+  // Communication with 1 physical DLL with using exported methods.
+  // Main targets: initialization of DLL, creation of ExtAIs and termination of DLL and ExtAIs
+  TCommDLL = class
+  private
+    fLibHandle: THandle;
+    fDLLPath: String;
+    fExtAIAPICnt: ui8; //@Martin: Use Delphi datatypes in Delphi please. Integer here

```

Comment:
In the game probably not (can be replaced by events). I just needed some function to test callbacks.